src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -232,7 +235,7 @@ private function addLongOption($name, $value)
             if (isset($next[0]) && '-' !== $next[0]) {
                 $value = $next;
             } elseif (empty($next)) {
-                $value = '';
+                $value = null;
```

Comment:
This change is needed, make the behavior consistent with [the expected one](https://github.com/symfony/symfony/pull/19946/files#diff-3ae4ec9ae7b044c86a061156979d87fbR222).